src/sql/workbench/contrib/notebook/browser/models/modelInterfaces.ts

```diff
@@ -263,7 +265,7 @@ export interface INotebookModel {
 	/**
 	 * The active cell for this model. May be undefined
 	 */
-	readonly activeCell: ICellModel;
+	activeCell: ICellModel;
```

Comment:
If this is possibly undefined as the comment suggests you should update this to be ICellModel | undefined